application/controllers/exercises.py

```diff
@@ -38,6 +40,16 @@
     help='category parameter is required'
 )
 
+class RandExercises(Resource):
+    def get(self):
+        category = request.json['category']
```

Comment:
smaahhhttt on the request.json! Duh that this is coming through in the request body!